### ISSUE TITLE
[Bug Report] Rewriting deleted records not working

### ISSUE THREAD

**Issue by MikaelGRA** [influxdata/influxdb#7117]
### Bug report
I am attempting to use the newly introduced "DELETE" operation.

If I first write data to a measurement, then delete a range of data in that measurement that includes the data I wrote, and then rewrite the data and attempt to query it again, no data is returned.

Is this the expected behaviour or a bug?

__System info:__
Influxdb 0.13 on Ubuntu

__Steps to reproduce:__

```
# create database
curl -G -X POST http://localhost:8086/query --data-urlencode "q=CREATE DATABASE mydb"

# write a record  to 'data' measurement
curl -X POST http://localhost:8086/write?db=mydb --data-binary "data value=61 1293843669000000000"

# read the record from the 'data' measurement (this works as expected)
curl -G http://localhost:8086/query?db=mydb --data-urlencode "q=SELECT * from data WHERE '2011-01-01' <= time AND time < '2012-01-01'"

# delete the record in the 'data' measurement by deleting a range of data
curl -G http://localhost:8086/query?db=mydb --data-urlencode "q=DELETE FROM data WHERE '2011-01-01' <= time AND time < '2012-01-01'"

# write a record  to 'data' measurement (again)
curl -X POST http://localhost:8086/write?db=mydb --data-binary "data value=61 1293843669000000000"

# read the record from the 'data' measurement (the record is not returned)
curl -G http://localhost:8086/query?db=mydb --data-urlencode "q=SELECT * from data WHERE '2011-01-01' <= time AND time < '2012-01-01'"
```

__Expected behavior:__ 
When I query the record after it has been rewritten, I expect that record to be returned.

__Actual behavior:__
The record is not returned.

**Comment by jsternberg** [comment]
@MikaelGRA can you try the latest beta? I think this problem may have already been fixed.

**Comment by MikaelGRA** [comment]
I would like to try that out. 

However, I am not very proficient when it comes to linux distribution.

Is there a guide for installing it, that is as clear as the one provided for 0.13?
https://docs.influxdata.com/influxdb/v0.13/introduction/installation/

**Comment by jsternberg** [comment]
@MikaelGRA https://docs.influxdata.com/influxdb/v1.0/introduction/installation/

**Comment by MikaelGRA** [comment]
That guide installs v. 0.13 though :)

**Comment by jsternberg** [comment]
You can just go to the downloads page and switch the tab to `1.0.0-beta3` to download the debian package: https://influxdata.com/downloads/#influxdb

Then just use `dpkg -i <filename>`.

**Comment by MikaelGRA** [comment]
Hello again,

I can confirm the problem is fixed on the latest version 1.0.0-beta3.

Thanks for your help. :)
